### ISSUE TITLE
Text dissapears from textfield

### ISSUE THREAD

**Issue by juanamd** [Gamua/Starling-Framework#483]
Hi Daniel,

First of all, i didn't test this enough because i don't have so much time atm, so maybe its something on my end.
Today i switched to the latest github build of Starling in my game, and noticed that some textfields where not appearing as before (previously i had a build near the release of 1.4).
I don't have the time right now to build a test case, but i can reproduce the issue like this:
1- Have a textfield with some text in it, placed on the stage: everything is fine.
2- Remove it from its parent and dispose it (it is using a bitmap font on a texture atlas)
3- Change the text by doing: _textfield.text = String(someInt);
4- Adding it to the stage again
5- Now the textfield is not displayed.

I found a workaroudn this: if in step 4 i do this instead: _textfield.text = String(someInt) + " ";
In other words, if i append some text, it now displays it correctly.
I think this is odd, since if i revert my previous Starling build, everything works fine again without the workaround, so thats why im filing it as a bug.

Thanks!
